Checkbox/Checkbox.vue

```diff
@@ -46,6 +46,30 @@ function handleStateChange() {
     )
   }
 }
+
+const checkboxClassVisuals = computed(() => {
```

Comment:
In general, can we move this logic into the `toggleState` computed? Seems kinda unnecessary to have a separate computed for this and also for the label below.

```
const toggleState = computed(() => {
  ...

  return {
    checked,
    indeterminate,
    labelClass, // <-- extend this with the `labelClass` prop
    checkboxClass
  }
})
```